client/packages/config/src/routes.ts

```diff
@@ -65,5 +65,23 @@ export enum AppRoute {
 }
 
 export enum ExternalURL {
-  PublicDocs = 'https://docs.msupply.foundation/docs',
+  // PublicDocs = 'https://docs.msupply.foundation/docs',
+  PublicDocs,
 }
+
+export interface LocalisedExternalUrlProps {
+  url: ExternalURL;
+  locale: String | undefined;
+}
+
+export const localisedExternalUrl = ({
```

Comment:
Rather than creating a new interface I would create a hook, then you don't need to pass in the locale

```
export const useExternalUrl = (url: ExternalURL) => {
  const { currentLanguage } = useIntlUtils();
  switch (url) {
    case ExternalURL.PublicDocs: {
      const baseUrl = 'https://docs.msupply.foundation/';
      switch (currentLanguage) {
        case 'es':
          return `${baseUrl}es/docs`;
        case 'fr':
        case 'fr-DJ':
          return `${baseUrl}fr/docs`;
        default:
          return `${baseUrl}docs`;
      }
    }
  }
};
```